fili-core/src/test/groovy/com/yahoo/bard/webservice/data/dimension/impl/LuceneSearchProviderSpec.groovy

```diff
@@ -191,6 +192,7 @@ class LuceneSearchProviderSpec extends SearchProviderSpec<LuceneSearchProvider>
         !Files.exists(file4)
     }
 
+    @Ignore("This test is currently not valid because the replacement index is invalid.")
```

Comment:
This shouldn't seem to be ignored, 'cause it was working previously

Reply:
It no longer works, because the file we are using is fake, the method now requires a real index file